# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/gcc/gcc33/fixincl.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./gcc/fixinc/fixincl.sh.orig	Sat May 11 16:11:15 2002
+++ ./gcc/fixinc/fixincl.sh	Sat May 11 16:12:00 2002
@@ -143,6 +143,8 @@
     INPUTLIST="$@"
 fi
 
+INPUTLIST=""
+
 for INPUT in ${INPUTLIST} ; do
 
 cd ${ORIGDIR}
